tinkerbell/boots/templates/deployment.yaml

```diff
@@ -59,10 +54,6 @@ spec:
               value: {{ printf "%s:%v" $tinkServerIp .Values.tinkServer.port | quote }}
             - name: TINKERBELL_TLS
               value: {{ .Values.tinkServer.tls | quote }}
-            - name: BOOTS_EXTRA_KERNEL_ARGS
```

Comment:
I believe this is still needed, no? https://github.com/tinkerbell/boots/blob/da970005705e1a670486933adc39b7a8b8c37d4c/cmd/boots/cli.go#L77

Reply:
But this is an env var, and that one is cli arg, right ?

Reply:
I see, alright will revert it back then ;)